recipes/genomedata/meta.yaml

```diff
@@ -0,0 +1,97 @@
+package:
+  name: genomedata
+  version: "1.3.5"
+
+source:
+  fn: genomedata-1.3.5.tar.gz
+  url: https://files.pythonhosted.org/packages/ae/c8/504b3674a150128e61f5cd075900cc2139e0ac48dabdb70ddf314fbe8424/genomedata-1.3.5.tar.gz
+  md5: b311a41a97c748ffff59c21b8a1d7ac8
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # noarch_python: True
+  preserve_egg_dir: True
+  skip: True # [not py27]
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - genomedata = genomedata:main
+    #
+    # Would create an entry point called genomedata that calls genomedata.main()
+
+    - genomedata-info=genomedata._info:main
+    - genomedata-query=genomedata._query:main
+    - genomedata-histogram=genomedata._histogram:main
+    - genomedata-load=genomedata.load_genomedata:main
+    - genomedata-load-seq=genomedata._load_seq:main
+    - genomedata-load-assembly=genomedata._load_seq:main
+    - genomedata-open-data=genomedata._open_data:main
+    - genomedata-close-data=genomedata._close_data:main
+    - genomedata-report=genomedata._report:main
+    - genomedata-erase-data=genomedata._erase_data:main
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - forked-path
+    - pytables >=3.0
+    - textinput
+    - gcc
+    - hdf5
+    - hgtools
+
+  run:
+    - python
+    - setuptools
```

Comment:
Probably not needed, good catch. I'll go through and remove those throughout.

For C-based packages that use gcc, are we normally requiring libgcc at the runtime? I wouldn't normally do that, but I've seen some discussion surrounding it here so I added it in.